src/mapnode.cpp

```diff
@@ -170,64 +170,77 @@ void MapNode::rotateAlongYAxis(const NodeDefManager *nodemgr, Rotation rot)
 	}
 }
 
+void buildFixedNodeBox(const MapNode &n, const NodeBox &nodebox,
+		const NodeDefManager *nodemgr,
+		std::vector<aabb3f> &boxes, const std::vector<aabb3f> &input_boxes,
+		NodeBoxType nbt) {
+	u8 facedir = n.getFaceDir(nodemgr, true);
+	u8 axisdir = facedir >> 2;
+	facedir &= 0x03;
+	const auto &fixed = nodebox.fixed;
+	boxes.reserve(boxes.size() + fixed.size());
```

Comment:
so `nodebox` is entirely unused except for retrieving the size and calling `reserve`